examples/barotropicqg_betaforced.jl

```diff
@@ -145,64 +148,81 @@ nothing # hide
-  sol, v, p, g = prob.sol, prob.vars, prob.params, prob.grid
-  BarotropicQG.updatevars!(prob)
-
-  sca(axs[1])
-  cla()
-  pcolormesh(x, y, v.q)
-  axis("square")
-  xticks(-3:1:3)
-  yticks(-3:1:3)
-  title(L"vorticity $\zeta = \partial_x v - \partial_y u$")
-  if drawcolorbar==true
-    colorbar()
-  end
-
-  sca(axs[2])
-  cla()
-  contourf(x, y, v.psi)
-  if maximum(abs.(v.psi))>0
-    contour(x, y, v.psi, colors="k")
-  end
-  axis("square")
-  xticks(-3:1:3)
-  yticks(-3:1:3)
-  title(L"streamfunction $\psi$")
-  if drawcolorbar==true
-    colorbar()
-  end
-
-  sca(axs[3])
-  cla()
-  plot(Array(transpose(mean(v.zeta, dims=1))), y[1,:])
-  plot(0*y[1,:], y[1,:], "k--")
-  ylim(-Lx/2, Lx/2)
-  xlim(-3, 3)
-  title(L"zonal mean $\zeta$")
-
-  sca(axs[4])
-  cla()
-  plot(Array(transpose(mean(v.u, dims=1))), y[1,:])
-  plot(0*y[1,:], y[1,:], "k--")
-  ylim(-Lx/2, Lx/2)
-  xlim(-0.5, 0.5)
-  title(L"zonal mean $u$")
-
-  sca(axs[5])
-  cla()
-  plot(μ*E.t[1:E.i], E.data[1:E.i], label="energy")
-  xlabel(L"\mu t")
-  legend()
-
-  sca(axs[6])
-  cla()
-  plot(μ*Z.t[1:Z.i], Z.data[1:E.i], label="enstrophy")
-  xlabel(L"\mu t")
-  legend()
+function plot_output(prob)
+  l = @layout grid(2, 3)
+  
+  pζ = heatmap(x, y, vs.zeta,
+            aspectratio = 1,
+            legend = false
+                 c = :balance,
+              clim = (-8, 8),
+             xlims = (-gr.Lx/2, gr.Lx/2),
+             ylims = (-gr.Ly/2, gr.Ly/2),
+            xticks = -3:3,
+            yticks = -3:3,
+            xlabel = "x",
+            ylabel = "y",
+             title = "vorticity ζ=∂v/∂x-∂u/∂y",
+        framestyle = :box)
+
+  pψ = contourf(x, y, vs.psi,
+                levels = -0.32:0.04:0.32,
+            aspectratio = 1,
+         linewidth = 1,
+            legend = false,
+              clim = (-0.22, 0.22),
+                 c = :viridis,
+             xlims = (-gr.Lx/2, gr.Lx/2),
+             ylims = (-gr.Ly/2, gr.Ly/2),
+            xticks = -3:3,
+            yticks = -3:3,
+            xlabel = "x",
+            ylabel = "y",
+             title = "streamfunction ψ",
+        framestyle = :box)
+
+  pζm = plot(mean(vs.zeta, dims=1)', y,
+          legend = false,
+          linewidth = 2,
+          alpha = 0.7,
+          yticks = -3:3,
+          xlims = (-3, 3),
+          xlabel = "zonal mean ζ",
+          ylabel = "y")
+  plot!(pζm, 0*y, y, linestyle=:dash, linecolor=:black)
+
+  pum = plot(mean(vs.u, dims=1)', y,
+          legend = false,
+          linewidth = 2,
+          alpha = 0.7,
+          yticks = -3:3,
+          xlims = (-0.5, 0.5),
+          xlabel = "zonal mean u",
+          ylabel = "y")
+  plot!(pum, 0*y, y, linestyle=:dash, linecolor=:black)
+
+  pE = plot(1,
+          label="energy",
+          linewidth = 2,
+          alpha = 0.7,
+          xlims = (-0.1, 4.1),
+          ylims = (0, 0.05),
+          xlabel = "μt")
+          
+  pZ = plot(1,
+          label="enstrophy",
```

Comment:
It's of course not a big deal, but there is a typo here: there are no spaces surrounding `=` in label, while there are in the rest.